pkg/edgeview/Dockerfile.query

```diff
@@ -0,0 +1,22 @@
+FROM lfedge/eve-alpine:6.7.0 as build
```

Comment:
Why do we need two separate Dockerfiles? Can't we conditionalize one somehow?

Reply:
the 2nd one is not for EVE device, it's for someone to cd to the directory and make a 'client' docker container.